include/deal.II/grid/grid_tools.h

```diff
@@ -454,6 +454,18 @@ namespace GridTools
     const std::vector<Point<spacedim>> &all_vertices,
     std::vector<CellData<dim>> &        cells);
 
+  /**
+   * This function is identical to invert_all_negative_measure_cells() except it
```

Comment:
add
"Check the given cells and inverts any cell that is considered to have negative measure/volume
in the orientation required by deal.II." (empty line)